Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -296,6 +299,8 @@ final class MongoCollectionTests: XCTestCase {
         for doc in try coll1.find() {
             expect(doc).to(beAnInstanceOf(Basic.self))
         }
+
+        // add tests here using findAndModify commands
```

Comment:
Outstanding TODO item? Should there be a ticket for this if it's not being done in this PR?